src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="main">
+      <div class="calendar calendar--month-length calendar--start-day--mon">
```

Comment:
i guess month length is not working, isn't it?
```suggestion
      <div class="calendar calendar--month-length--29 calendar--start-day--mon">
```

Reply:
Yes, it didn't work, but I fixed iti fixed it